integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -174,19 +191,31 @@ public void lockBlockTest4() throws IOException {
       fileIds.add(TachyonFSTestUtils.createByteFile(mTfs, uniqPath + k, WriteType.CACHE_THROUGH,
           fileSize));
     }
+
     for (int k = 0; k <= numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
       CommonUtils.sleepMs(null, getSleepMs());
       Assert.assertFalse(tFile.isInMemory());
+      is = tFile.getInStream(ReadType.NO_CACHE);
+      buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
       if (k < numOfFiles) {
-        Assert.assertNull(tFile.readByteBuffer(0));
+        int r = is.read(buf.array());
+        Assert.assertEquals(-1, r);
         Assert.assertTrue(tFile.recache());
-        Assert.assertNotNull(tFile.readByteBuffer(0));
+        is.seek(0);
+        buf.clear();
+        r = is.read(buf.array());
+        Assert.assertTrue(r != -1);
       } else {
-        Assert.assertNull(tFile.readByteBuffer(0));
+        int r = is.read(buf.array());
+        Assert.assertTrue(r == -1);
```

Comment:
This can be combined with the previous line.

Also, this can be assertEquals()?